cmd/orb_test.go

```diff
@@ -2581,4 +2581,69 @@ https://circleci.com/orbs/registry/orb/my/orb
 			})
 		})
 	})
+
```

Comment:
Since all of our Go tests have this really painful format but are fundamentally just doing "run the thing with some conditions, check the output," we've been using (and really enjoying) cucumber for this, for example: https://github.com/CircleCI-Public/circleci-cli/blob/master/integration_tests/features/contexts.feature

I'd highly recommend writing these as cucumber tests, they should only be a few lines each.

Reply:
We should have tests for the failure cases too - e.g. what happens when `@orb.yml` doesn't exist, what happens when an included file doesn't exist, etc.